chore(blog): add April summary (#4055)
asyncapi/website: markdown/blog/2025-april-summary.md

---
title: "Monthly Community Update: April 2025"
date: 2025-04-30T06:00:00+01:00
type: Communication
tags:
  - Project Status
cover: /img/posts/2025-blog-banner/banner-april.webp
authors:
  - name: Thulisile Sibanda
    photo: /img/avatars/thulieblack.webp
    link: https://www.linkedin.com/in/v-thulisile-sibanda/
    byline: Community Builder and Open Source Fanatic!
excerpt: 'AsyncAPI community status and project updates for April 2025'
featured: true
---

We had a busy month, from having our first conference of the year to receiving 204 applications for Google Summer of Code (GSoC'25). Let's dive into the April update!

## AsyncAPI Governance Board 

> The vote for the new AsyncAPI Governance Board is now open. Part of the change will be to replace the single Executive Director position and have the option of Ambassadors becoming TSC members. 

Reviewer: ```suggestion
The vote to introduce the AsyncAPI Governance Board is now open. Part of the change will be replacing the Executive Director position. Additionally, Ambassadors will have the right to be part of the TSC. 
```

---
title: "Monthly Community Update: April 2025"
date: 2025-04-30T06:00:00+01:00
type: Communication
tags:
  - Project Status
cover: /img/posts/2025-blog-banner/banner-april.webp
authors:
  - name: Thulisile Sibanda
    photo: /img/avatars/thulieblack.webp
    link: https://www.linkedin.com/in/v-thulisile-sibanda/
    byline: Community Builder and Open Source Fanatic!
excerpt: 'AsyncAPI community status and project updates for April 2025'
featured: true
---

We had a busy month, from having our first conference of the year, receiving 204 applications for Google Summer of Code (GSoC'25),  to some sponsorship updates. Let's dive into the April update!

## New Sponsors

> We are excited to share that [Mercedes-Benz](https://opensource.mercedes-benz.com/manifesto/) is supporting the initiative as our Bronze sponsor and, most importantly, that they have officially confirmed they use AsyncAPI.

Reviewer: the prospectus do not offer sharing such info about bronze sponsors.

so only social media posts.

cc @iambami 

<img width="953" alt="Screenshot 2025-04-23 at 09 56 20" src="https://github.com/user-attachments/assets/b2c5e3c3-5ba5-44c1-a02a-cdb817344583" />

Author: So I must remove the new sponsors section then?

Reviewer: yes, we should follow the rules we setup :)

Reviewer: and same with GitHub and RedHat

Author: 🫡🫡

<a href='https://opensource.mercedes-benz.com/' target='_blank'>
<img src='https://github.com/asyncapi/spec/blob/master/assets/mercedes-benz.png?raw=true' alt='Mercedes-Benz' width='500px' />
</a>

Additionally, [APIDeck](https://www.apideck.com/) has upgraded sponsorship to Bronze; a huge thank you to [Samir Amzani](https://www.linkedin.com/in/amzani) for the support and effort.

<a href='https://www.apideck.com/' target='_blank'>
<img src='https://github.com/asyncapi/spec/blob/master/assets/apideck.png?raw=true' alt='APIDeck' width='500px' />
</a>

> ## Sponsorship Updates

Reviewer: yeah, same like with the other comment - we should not have it here

While submitting your proposals, please select the "**AsyncAPI**" option for the track to ensure your submission goes to the correct category.

### Call for Sponsors
We are still open to financial sponsors for the AsyncAPI Conf 2025. [Please help us by sharing the conference prospectus within your networks](https://drive.google.com/file/d/1LxTFLWkRmFQkHOyrwZBL7yqSO2_XPJNq/view?usp=sharing).

## Spec x Tooling 
[Godwin Alexander](https://github.com/oviecodes) recently opened a discussion about the future of [Glee](https://github.com/asyncapi/glee), noting that it is no longer being actively maintained. He is seeking input from the community on whether we should continue to invest in Glee. [Please share your thoughts in the Future of Glee discussion](https://github.com/orgs/asyncapi/discussions/1861).

> ## Onboarding/Training Videos

Reviewer: I think we have option to embed videos, you should try this for this video, to embed in article

Author: AAAH I had forgotten about that, I'll add the embed
